vmdb/app/views/support/show.html.haml

```diff
@@ -55,5 +55,6 @@
               - rh_cust_portal = link_to(_("Red Hat Customer Portal"),
                                          "https://access.redhat.com/home",
                                          :onclick => "return miqClickAndPop(this);")
-              =_("For questions or problem reporting, go to the %s") % rh_cust_portal
+              = _("For questions or problem reporting, go to the")
```

Comment:
The other PRs I've been reading seem to encourage the `%s` form.
Do we have a guideline?